### ISSUE TITLE
Create Classifier Evaluation Function and Run for All Classifiers

### ISSUE THREAD

**Issue by ijyliu** [ijyliu/computer-vision-project#52]
Write a function in a .py file that we can use for evaluation. Then write evaluation notebooks for each classifier that call this.

Input: Filepath for Classifier Predictions Excel File, Classifier Name

Output:

- Accuracy, Precision, Recall, etc. (Sklearn classification report https://scikit-learn.org/stable/modules/generated/sklearn.metrics.classification_report.html but saved as an Excel file is good)
- Confusion matrix as png
- Plot of a Row of 5 randomly selected Misclassified Images (subplot). The title over each image should be true_class + "Misclassified As" + predicted_class. you should use the no blur filepath to load the images. save as png.

**Comment by ijyliu** [comment]
you should also remove all underscores from the plot title, everything should be formatted neatly because they care about presentation

**Comment by ijyliu** [comment]
@ashutoshtiwari13 

- which model are the misclassified images from. this needs to be clear
- please fix the underscores in the confusion matrix plot titles and make them look professional. these need to be neat, they are grading for plot presentation

**Comment by ijyliu** [comment]
completed by @ijyliu, @ashutoshtiwari13 in default of obligation
